packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/ToggleSwitch.tsx

```diff
@@ -72,12 +83,10 @@ export const ToggleSwitch = ({
 		return !!defaultChecked;
 	};
 
-	const isSlim = size === 'slim';
-
 	return (
 		<div css={[toggleSwitchStyles, cssOverrides]} {...props}>
 			<button
```

Comment:
i know it's not part of this pr, so don't let this block merging, but does anyone know why this is a button and not a checkbox?

Reply:
@marjisound do you know about this one maybe?

Reply:
That's right, it's a button but with a `role="switch"` and it's using aria-checked & aria-labelledby for accessibility. 

Back when I was implementing this component, @SiAdcock sent me a blog (inclusive-components) as I wanted to get some ideas on accessibility and in there, there is this page for toggles which is taking the button approach for the toggle https://inclusive-components.design/toggle-button/

That's why I decided to use button. @sndrs  Do you think using a `<input type="checkbox">` would be more beneficial?

Reply:
I still think this is fine (in Heydon I trust!) but let me know if you have any thoughts @sndrs 🙂

Reply:
no no, was just curious. if this is best way i'm good (and til 😄)